c10/core/CPUAllocator.cpp

```diff
@@ -36,7 +36,7 @@ void memset_junk(void* data, size_t num) {
   }
 }
 
-void* alloc_cpu(size_t nbytes) {
+void* alloc_cpu(size_t nbytes, const size_t alignment) {
```

Comment:
Must either modify this function to accept alignment as input, or copy-paste it with this addition.  I decided to go with modifying this function, defaulting the second argument to gAlignment.  Any issues with this?  API stability and backwards compatibility, etc.?

Reply:
It looks like this argument will always be 16 on mobile and 64 on non-mobile.  Maybe we should just change the definition of gAlignment to reflect that and then we don't need this part of the change?

Reply:
That'll push the #ifdef ARM to the header, are we OK with that?  Alternatively we can move gAlignment to the .cpp file.

Reply:
Done. Marking as resolved.